2024-09-27 13:18:32,323 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-09-27 13:18:32,323 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\mahmo\OneDrive\Documents\workspace\Web\instapy_env\Lib\site-packages\instabot\bot\bot.py
2024-09-27 13:18:32,323 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-09-27 13:18:32,584 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-09-27 13:18:32,821 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-09-27 13:18:33,204 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-09-27 13:18:33,453 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-09-27 13:18:34,834 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-09-27 13:18:34,834 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXFFs14_rWZ8W1p1eVt9yaJpyndY7z77vyQyhKiLy4tqyvw9W4PWmIXmtLNnzUltdtJCbOw/AfywT7Bhb7VYWWToq-eOzwlOFpjtgcLk5pQEORvR_i0mWoYVa-EY_NN0gJ-8pfnyO6dBd1OsV2ys9Q/","api_path":"/challenge/AXFFs14_rWZ8W1p1eVt9yaJpyndY7z77vyQyhKiLy4tqyvw9W4PWmIXmtLNnzUltdtJCbOw/AfywT7Bhb7VYWWToq-eOzwlOFpjtgcLk5pQEORvR_i0mWoYVa-EY_NN0gJ-8pfnyO6dBd1OsV2ys9Q/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2024-09-27 13:18:34,834 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-09-27 13:18:34,834 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2024-09-27 13:18:34,834 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2024-09-27 13:18:51,160 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-09-27 13:18:51,176 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\mahmo\OneDrive\Documents\workspace\Web\instapy_env\Lib\site-packages\instabot\bot\bot.py
2024-09-27 13:18:51,177 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-09-27 13:18:51,422 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-09-27 13:18:51,706 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-09-27 13:18:52,043 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-09-27 13:18:52,354 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-09-27 13:18:53,602 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-09-27 13:18:53,602 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXFiw0KGaoqPrNNbwZKWJrcVwX5DNkSMG5P9hKrdt8DAdKRyVZL33LRY480fmU0RePAkfPg/Afxnp5HfzZTpkxFbsKDeZKURtHsSsP3xDQd3XY_8dWOswy_zpZho4Sohz4ZCbDzhX-TeBjKcCTY1Ew/","api_path":"/challenge/AXFiw0KGaoqPrNNbwZKWJrcVwX5DNkSMG5P9hKrdt8DAdKRyVZL33LRY480fmU0RePAkfPg/Afxnp5HfzZTpkxFbsKDeZKURtHsSsP3xDQd3XY_8dWOswy_zpZho4Sohz4ZCbDzhX-TeBjKcCTY1Ew/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2024-09-27 13:18:53,602 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-09-27 13:18:53,602 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2024-09-27 13:18:53,611 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
